lib/patch-series.ts

```diff
@@ -718,9 +740,35 @@ export class PatchSeries {
             if (footers.length > 0) {
                 footers.push(""); // empty line
             }
-            // split the range-diff and prefix with a space
-            footers.push(`Range-diff vs v${this.metadata.iteration - 1}:\n\n${
-                         this.rangeDiff.replace(/(^|\n(?!$))/g, "$1 ")}\n`);
+
+            if (typeof this.options.doRangeDiff === "undefined" || this.options.doRangeDiff) {
```

Comment:
Could we do it the other way round? I do not feel very strongly about it, but I did stumble over this. The order "if it is set to something true-ish, or if it is not set at all" would have helped me.

Reply:
Need to know if present before checking a boolean.  Awkward so I  am changing the if/else order (even though it is now a string and I prefer the more positive side of the if first).